extensions/search-mdn/src/index.tsx

```diff
@@ -0,0 +1,113 @@
+import {
+  ActionPanel,
+  CopyToClipboardAction,
+  Detail,
+  Icon,
+  List,
+  OpenInBrowserAction,
+  showToast,
+  ToastStyle,
+  useNavigation,
+} from "@raycast/api";
+import { useState, useEffect } from "react";
+import axios from "axios";
+import urljoin from "url-join";
+
+export default function MDNSearchResultsList() {
+  const [query, setQuery] = useState<null | string>(null);
+  const [state, setState] = useState<Result[]>([]);
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+  const { push } = useNavigation();
+
+  useEffect(() => {
+    async function fetch() {
+      if (query) {
```

Comment:
Let's proceed with the case with the empty query as well, because if I will enter something in the searchbar and then remove it – old results stay, which is not the consistent behaviour with other extensions and Raycast in general.

![CleanShot 2021-10-20 at 21 43 02@2x](https://user-images.githubusercontent.com/15333662/138152598-9b692901-8647-4433-b720-00f982bb78c6.png)

You can just do `setState([])` here as workaround.